pkg/costmodel/metrics.go

```diff
@@ -232,6 +234,22 @@ func initCostModelMetrics(clusterCache clustercache.ClusterCache, provider model
 			toRegisterGV = append(toRegisterGV, pvAllocGv)
 		}
 
+		ramIdleGv = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "container_ram_idle",
```

Comment:
This is a fine metric to measure. Unfortunately, there may be confusion caused by the use of the word "idle" in other contexts---most frequently to describe node resources that are not allocated to a workload. (Though sometimes we do just qualify the type of idle, as in "workload idle," and accept the fact that there can be multiple kinds of idle, and multiple kinds of efficiency.)

Do you have an opinion here @kwombach? or @AjayTripathy and @dwbrown2 ?